pkg/deployments/README.md

```diff
@@ -57,28 +57,16 @@ Returns the address of a contract's canonical deployment.
 
 Returns an object with all contracts from a deployment and their addresses.
 
-## Past Deployments
+## Active Deployments
 
 | Description                                          | Task ID                                                                                              |
 | ---------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
 | Authorizer, governance contract                      | [`20210418-authorizer`](./tasks/20210418-authorizer)                                                 |
 | Vault, main protocol contract                        | [`20210418-vault`](./tasks/20210418-vault)                                                           |
 | Weighted Pools of up to 8 tokens                     | [`20210418-weighted-pool`](./tasks/20210418-weighted-pool)                                           |
-| Weighted Pools with two tokens and price oracle      | [`20210418-weighted-pool`](./tasks/20210418-weighted-pool)                                           |
```

Comment:
Would you say this is deprecated right now? afaik we use this on the frontend when deploying a 2 token pool atm.

Reply:
I know that oracles are not being maintained going forwards but there's no reason to deprecate this deployment right at the minute.

Reply:
I mean the pools are fine in the sense that they are not incorrect, but discouraging people from creating more of them seems to be in the spirit of what we're doing.

Reply:
Fair, I'd say that if we're actively allowing new pools to be created on the frontend then we haven't deprecated it though. If we wanted to do that then we should stop the frontend from interacting with this factory surely?

Reply:
Yes - this PR is only part of the effort :stuck_out_tongue: